### DISCUSSION TITLE
Websites

### DISCUSSION THREAD

**Discussion by rkaminsk** [potassco/python-project-template#12] [Ideas]
There should be some standard way to provide a webpage for a project. Candidates are:

- readthedocs
- github pages
- our potassco page

For small projects readthedocs should be sufficient to provide a webpage. Linking it on potassco.org would be helpful.

**Comment by MaxOstrowski** [comment]
Sounds good to me. It would be cool if at least all new and still active projects could then align at some point if possible without too much hassle.
